src/styles/main.scss

```diff
@@ -14,3 +14,7 @@ span:hover {
   font-weight: 700;
   cursor: pointer;
 }
+
+.hidden {
+  display: none;
+}
```

Comment:
this task was meant to be done without changing CSS styles
you can use 'hidden' attribute instead of separate class